huaweicloud/services/acceptance/dew/data_source_huaweicloud_kms_key_tags_test.go

```diff
@@ -0,0 +1,36 @@
+package dew
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDataSourceKmsKeyTags_basic(t *testing.T) {
+	var (
+		dataSourcename = "data.huaweicloud_kms_key_tags.test"
+		dc             = acceptance.InitDataSourceCheck(dataSourcename)
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
```

Comment:
This test case cannot guarantee success every time. It is recommended to add environment variables.